src/EA.Weee.Web/Views/OrganisationRegistration/PartnershipDetails.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using EA.Weee.Core.DataStandards
+@using EA.Weee.Web.Areas.Admin.Controllers
+@using MvcNavigationHelpers
+@using EA.Weee.Web.Controllers
+@model EA.Weee.Web.Areas.Admin.ViewModels.AddOrganisation.Details.PartnershipDetailsViewModel
+@{
+    ViewBag.Title = "Partnership details";
+}
+@section main_content
```

Comment:
There is a new section full_main_content that provides a wider screen. Think we use that for these new screens.